### DISCUSSION TITLE
Why is the default value of monai sliding window inference sigma_scale is 0.125?

### DISCUSSION THREAD

**Discussion by rishabhsharma22** [Project-MONAI/MONAI#5923] [General]
I have been playing around with monai to get an idea and feel of how the tool works. I learned that the sigma_scale argument for the sliding window inference is very important when we use the gaussian mode of blending the windows. However, I would like to know if there was a scientific reason behind setting the default value to 0.125 for the sigma scale and what will be the difference of using this sigma_Scale for a window size of 64 vs a window size of 128 not visually on the images but the theoretical reasoning behind the default sigma_scale argument value.

**Comment by KumoLiu** [comment]
Hi @rishabhsharma22, thanks for your interest here. IMO when blending overlapping windows, gaussian mode actually means to assign smaller weights to the edges. The edges' lesser weight is also a result of the possibility that the segmentation of the edges during sliding window inference is not very precise. The sigma scale  is actually the standard variance coefficient of guassian in this case. The weight assigned to the edges will increase as this value rises.

Additionally, because sigma only impacts the window's edges, the value of sigma does not directly correlate with the size of the window.
<img width="1007" alt="Screen Shot 2023-02-01 at 11 17 22" src="https://user-images.githubusercontent.com/55491388/215936824-2e0a0d31-ab90-401a-80a6-7467e6f93948.png">
Hope it can help you, thanks!

**Reply by rishabhsharma22** [reply]
Hello @KumoLiu,

I guess this visualization has definitely helped me a little better to understand what is happening with the model. A question, I believe that the dark purple pixels have a value of 0, that is the gaussian multiplier is 0. Am I correct?

**Reply by KumoLiu** [reply]
Yes, just add colorbar to help you understand.
<img width="496" alt="Screen Shot 2023-02-01 at 11 38 54" src="https://user-images.githubusercontent.com/55491388/215939419-e5df420e-9fea-4423-a0ad-51704bc9930f.png">
